test_models/valids/index.ts

```diff
@@ -11,6 +11,7 @@ const istandard = require('./istandard.json');
 const person = require('./person.json');
 const partWhole = require('./partWhole.json');
 const annotations = require('./annotations.json');
+const packages = require('./packages.json');
```

Comment:
I added this new test model to isolate the tests related to multi-package transformation.

Reply:
Root package name fixed!